packages/react/src/blocks/Symbol.tsx

```diff
@@ -136,7 +136,11 @@ class SymbolComponent extends React.Component<SymbolProps> {
                   context={{ ...state.context }}
                   modelName={model}
                   entry={entry}
-                  data={this.props.inheritState ? { ...data, ...state.state } : data}
+                  data={{
+                    ...data,
+                    ...(!!this.props.inheritState && state),
```

Comment:
I think this needs to be `state.state`

Reply:
(bad name I know, since `state` refers to the react internal context state, which has `context` which is the Builder context, and `state` which is the Builder state (and other stuff like `content` that is Builder content, etc))